demisto_sdk/commands/content_graph/strict_objects/base_strict_model.py

```diff
@@ -0,0 +1,179 @@
+from abc import ABC
+from typing import Any, List, Optional, Type, Union
+
+import pydantic
+from pydantic import BaseModel, Extra, Field, validator
+from pydantic.fields import FieldInfo
+
+from demisto_sdk.commands.common.constants import (
+    TYPE_JS,
+    TYPE_PWSH,
+    TYPE_PYTHON,
+    Auto,
+    MarketplaceVersions,
+)
+from demisto_sdk.commands.common.StrEnum import StrEnum
+
+marketplace_suffixes = [marketplace.value for marketplace in MarketplaceVersions]
+
+
+class BaseStrictModel(BaseModel, ABC):
+    class Config:
+        """
+        This is the definition of not allowing extra fields except those defined by the schema.
+        """
+
+        extra = Extra.forbid
+
+    @validator("*")
+    def prevent_none(cls, v):
+        """
+        Validator ensures no None value is entered in a field.
```

Comment:
elaborate on telling the difference between an empty and missing field, consider linking to that stackoverflow page to make it extra clear for future maintainers